apps/console/src/public/deployment.config.json

```diff
@@ -885,6 +885,16 @@
                     ]
                 }
             },
+            "policyAdministration":{
+                "disabledFeatures": [],
+                "enabled": false,
+                "scopes": {
+                    "create": [],
+                    "delete": [],
+                    "read": [],
+                    "update": []
```

Comment:
When the feature is onboarded, make sure to add the relevant scopes for the feature